<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="robots" content="index, follow, all" />
        <title>PropelException | Propel API</title>
                    <link rel="stylesheet" type="text/css" href="stylesheet.css">
                            </head>
        <body id="class">
            <div class="header">
        <ul>
            <li><a href="classes.html">Classes</a></li>
                        <li><a href="interfaces.html">Interfaces</a></li>
            <li><a href="doc-index.html">Index</a></li>
            
        </ul>

        <div id="title">Propel API</div>

            <div class="type">Class</div>
    <h1><a href=".html"></a>PropelException</h1>
    </div>
        <div class="content">
                <p>class    <strong>PropelException</strong>        extends <a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a></p>

            <div class="description">
            <p>The base class of all exceptions thrown by Propel.</p>
            <p></p>
        </div>
    
    
    
            <h2>Methods</h2>

            <table>
                    <tr>
                <td class="type">
                    
                </td>
                <td class="last">
                    <a href="#method___construct">__construct</a>($message = null, <a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a> $previous = null)
                    <p>Emulates wrapped exceptions for PHP &lt; 5.3</p>
                </td>
                <td></td>
            </tr>
                    <tr>
                <td class="type">
                    <a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a>
                </td>
                <td class="last">
                    <a href="#method_getCause">getCause</a>()
                    <p>Get the previous Exception We can&#039;t override getPrevious() since it&#039;s final</p>
                </td>
                <td></td>
            </tr>
            </table>


        <h2>Details</h2>

                        <h3 id="method___construct">
        <div class="location">at line 25</div>
        <code>            public            
    <strong>__construct</strong>($message = null, <a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a> $previous = null)</code>
    </h3>
    <div class="details">
        <p>Emulates wrapped exceptions for PHP &lt; 5.3</p>
        <p></p>
        <div class="tags">
                            <h4>Parameters</h4>

                    <table>
                    <tr>
                <td></td>
                <td>$message</td>
                <td></td>
            </tr>
                    <tr>
                <td><a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a></td>
                <td>$previous</td>
                <td></td>
            </tr>
            </table>

            
            
            
                    </div>
    </div>

                <h3 id="method_getCause">
        <div class="location">at line 51</div>
        <code>            public            <a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a>
    <strong>getCause</strong>()</code>
    </h3>
    <div class="details">
        <p>Get the previous Exception We can&#039;t override getPrevious() since it&#039;s final</p>
        <p></p>
        <div class="tags">
            
                            <h4>Return Value</h4>

                    <table>
        <tr>
            <td><a href="http://php.net/Exception"><abbr title="Exception">Exception</abbr></a></td>
            <td>The previous exception</td>
        </tr>
    </table>

            
            
                    </div>
    </div>

    
            </div>
            <div id="footer">
        Generated by <a href="http://sami.sensiolabs.org/" target="_top">Sami, the API Documentation Generator</a>.
    </div>
    </body>
</html>
